### ISSUE TITLE
check registration email

### ISSUE THREAD

**Issue by MisterMCman** [MisterMCman/talentengine#61]
When a user registers the backend sends an email. In the frontend a success message is shown which brings the user to the login form. The user can not login until he accepts the email. Check with Chris if the email server from the backend is online again.

**Comment by MisterMCman** [comment]
Can we use email for login instead of username? @PMVyatkin 
We could drop forgot username then.

**Comment by emochris** [comment]
No, only username is used for that because its the only unique element of a user. With the same eMail you can add many user-profiles.
